.gitattributes

```diff
@@ -7,3 +7,4 @@ tests/mock_triton_server/payloads/** filter=lfs diff=lfs merge=lfs -text
 tests/tests_data/** filter=lfs diff=lfs merge=lfs -text
 examples/basic_usage/img/** filter=lfs diff=lfs merge=lfs -text
 docs/source/img/* filter=lfs diff=lfs merge=lfs -text
+list filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why was this added?

Reply:
@bsuryadevara ?

Reply:
I'm not sure how that got into the file; possibly while I was trying to fix a gitlfs issue on the machine, that might have happened. It's not required. thanks